### PR TITLE
Challenge/edgar alandete superheroes routing

### PR DESCRIPTION
**Pull request by edalandete**

### CODE REVIEW

File: Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import DashboardComponent from './Components/DashboardComponent';
+
+function App() {
+  return (
+    <>
+      <Router>
+        <HeaderComponent />
+        <Switch>
+          <Route exact path="/dashboard" component={DashboardComponent} />
+          <Route exact path="/heroes/:heroId" component={HeroDetailComponent} />
```
Comment (albertmasats) [inline Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx:21]: Like you said to me the more specific one have to be the last. :)

**Review by albertmasats — APPROVED** [review]

**Review by cristian-bars — APPROVED** [review]
Good job!

File: Edgar-Alandete/challenges/superheroes-routing/superheroes/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```
Comment (gteixi) [inline Edgar-Alandete/challenges/superheroes-routing/superheroes/src/index.jsx:3]: el css siempre es el ultimo fichero a importar!

**Review by gteixi — APPROVED** [review]

File: Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './App.css';
```
Comment (davzlum) [inline Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx:2]: CSS lo último

**Review by IrenePerezAlonso — APPROVED** [review]

**Review by davzlum — APPROVED** [review]
Bien hecho!

File: Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx (hunk 3 above)
Comment (edalandete) [reply Edgar-Alandete/challenges/superheroes-routing/superheroes/src/App.jsx:2]: me lo apunto!!!!

**Pull request merged**